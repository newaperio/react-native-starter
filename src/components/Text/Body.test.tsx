import React from "react"
import renderer from "react-test-renderer"

import { Body } from "."

test("renders correctly", () => {
  const tree = renderer.create(<Body>Snapshot</Body>).toJSON()
  expect(tree).toMatchSnapshot()
})
