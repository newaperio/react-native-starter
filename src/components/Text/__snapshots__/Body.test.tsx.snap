// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#02181a",
      "fontFamily": "DM Sans",
    }
  }
>
  Snapshot
</Text>
`;
